LoopKit/TherapySetting.swift

```diff
@@ -0,0 +1,74 @@
+//
+
+
+public enum TherapySetting: Int {
+    case glucoseTargetRange
+    case correctionRangeOverrides
+    case suspendThreshold
+    case basalRate
+    case deliveryLimits
+    case insulinModel
+    case carbRatio
+    case insulinSensitivity
+    case none
+}
+
+public extension TherapySetting {
+    
+    // The following comes from https://tidepool.atlassian.net/browse/IFU-24
+    var title: String {
+        switch self {
+        case .glucoseTargetRange:
+            return LocalizedString("Correction Range", comment: "Title text for glucose target range")
+        case .correctionRangeOverrides:
+            return LocalizedString("Temporary Correction Ranges", comment: "Title text for temporary correction ranges")
+        case .suspendThreshold:
+            return LocalizedString("Suspend Threshold", comment: "Title text for suspend threshold")
+        case .basalRate:
+            return LocalizedString("Basal Rate", comment: "Title text for basal rate")
+        case .deliveryLimits:
+            return LocalizedString("Delivery Limits", comment: "Title text for delivery limits")
+        case .insulinModel:
+            return LocalizedString("Insulin Model", comment: "Title text for insulin model")
+        case .carbRatio:
+            return LocalizedString("Carb Ratio", comment: "Title text for carb ratio")
+        case .insulinSensitivity:
+            return LocalizedString("Insulin Sensitivities", comment: "Title text for insulin sensitivity")
+        case .none:
+            return ""
+        }
+    }
+    
+    var descriptiveText: String {
+        switch self {
+        case .glucoseTargetRange:
+            return LocalizedString("Correction range is the glucose value (or range of values) that you want Tidepool Loop to aim for in adjusting your basal insulin.", comment: "Descriptive text for glucose target range")
+        case .correctionRangeOverrides:
+            return LocalizedString("Pre-Meal and Workout ranges temporarily adjust your glucose target based on your selected activity.", comment: "Descriptive text for correction range overrides")
+        case .suspendThreshold:
+            return LocalizedString("When your glucose is predicted to go below this value, the app will recommend a basal rate of 0 U and will not recommend a bolus.", comment: "Descriptive text for suspend threshold")
+        case .basalRate:
+            return LocalizedString("Your basal rate of insulin is the number of units per hour that you want to use to cover your background insulin needs.", comment: "Descriptive text for basal rate")
+        case .deliveryLimits:
+            return LocalizedString("""
+                Maximum basal rate is the highest temporary basal rate Tidepool Loop is allowed to set automatically.
+
+                Maximum bolus is the highest bolus amount you can deliver at one time to cover carbs or bring down high glucose.
+            """, comment: "Descriptive text for delivery limits")
```

Comment:
Seems like the max basal rate and max bolus descriptions should be extracted from DeliveryLimitsEditor and unioned to form this string. WDYT?

Reply:
In the information screen, these two sentences are separated by a title, so there's a need for having them be separate strings if that's fine with you @rickpasetto

Reply:
The purpose of `descriptiveText` is to provide "official copy" (from IFU-24) for what each of these settings mean.  If any screen needs to add additional copy, that's fine, but the screen should do it on its own.

@mpangburn I haven't touched `DeliveryLimitsEditor` yet because I believe @novalegra is working on that screen right now.  Right, Anna?

Reply:
Yes, I am. If you want to go ahead and make the changes @rickpasetto, I'm happy for your changes to land first and me to pick them up in my open PR for LOOP-1488

Reply:
Roger that @novalegra, and thanks.  I'm going to move the `DeliveryLimits` struct to `LoopKit` too, to support your request.  I'll put that into this PR...stay tuned.